database/sqlite/sqlite_db_migration.c

```diff
@@ -214,6 +213,63 @@ static int do_migration_v7_v8(sqlite3 *database, const char *name)
     return 0;
 }
 
+static int do_migration_v8_v9(sqlite3 *database, const char *name)
+{
+    UNUSED(name);
+    info("Running database migration %s", name);
+
+    char sql[2048];
+    int rc;
+    sqlite3_stmt *res = NULL;
+
+    //create the health_log table and it's index
+    snprintfz(sql, 2047, "CREATE TABLE IF NOT EXISTS health_log (host_id blob, unique_id int, alarm_id int, alarm_event_id int, " \
+              "config_hash_id blob, updated_by_id int, updates_id int, when_key int, duration int, non_clear_duration int, " \
+              "flags int, exec_run_timestamp int, delay_up_to_timestamp int, name text, chart text, family text, exec text, " \
+              "recipient text, units text, info text, exec_code int, new_status real, old_status real, delay int, " \
+              "new_value double, old_value double, last_repeat int, chart_context text, " \
+              "transition_id blob, global_id int);");
```

Comment:
Only for we do not forget, but as we (@stelfrag , @MrZammler and me) already talked in the past different times, this table is necessary but it not sufficient. I cannot see atomicity in this table and consiering that neither primary key was made, so there are margin to improve. :thinking:

Reply:
Ok as also discussed with @stelfrag , I'll do an effort to break `health_log` into another table, `health_log_details`. The aim will be to keep basically one row per alert in `health_log`, having events go to the `health_log_details`.

Items such as `chart`, `name`, `chart_context`, `units`, etc will remain in `health_log`, while items like `new_value`, `old_value`, `statuses`, etc to go to `health_log_details`, thus minimizing repeating items. @thiagoftsm how does this sound?

Reply:
Hello @MrZammler ,

I understand what you wrote as a good step in the right direction . And I hope before migration 10 we can have a table for `host`, another for `alerts` and finally a relationship `m <-> n` for them. :)